api/services/elastigroup/aws/paths/ecsGroupClusterRollId.yaml

```diff
@@ -7,7 +7,14 @@ post:
   - "Elastigroup AWS"
   parameters:
     - $ref: "../../../../commons/parameters/accountId.yaml"
-    - $ref: "../../commons/parameters/groupId.yaml"
+    - in: path
```

Comment:
@vlad666spot why not using the reference?

Reply:
fixed